src/modules/ui/sidebar/selectors.ts

```diff
@@ -35,25 +40,32 @@ export const getSideBarCategories = createSelector<
   RootState,
   string | null,
   string,
+  string | null,
+  SidebarView,
   AssetPackState['data'],
   AssetState['data'],
   Category[]
 >(
   getSelectedAssetPackId,
   getSearch,
+  getSelectedCategory,
+  getSidebarView,
   getAssetPacks,
   getAssets,
-  (selectedAssetPackId, search, assetPacks, assets) => {
+  (selectedAssetPackId, search, category, view, assetPacks, assets) => {
+    assets = addNoGroundAsset(assets)
+    assetPacks = addNoGroundAssetPack(assetPacks)
```

Comment:
I could read the method definition but these two methods stand out to me. I'm not sure what it is they're doing and looking at the name/params I can't take a guess